### PR TITLE
align settings

### PR DESCRIPTION
**Pull request by MP91**
## Describe your changes
Align the devcontainer settings with the other repos

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,4 +4,10 @@ deprecation>=2.1.0
 pytest>=2.7.2
 PyYAML>=5.1
 rdflib>=6.0.0
-pre-commit>=2.19.0
+pre-commit
+pydocstyle
```
Comment (kse3hi) [inline requirements.txt:8]: Remove pydocstyle, it will be installed indirectly with flake8

File: requirements.txt [hunk 2]
```diff
@@ -4,4 +4,10 @@ deprecation>=2.1.0
...
+black
+pylint
```
Comment (kse3hi) [inline requirements.txt:10]: Remove pylint, as we going to not use it anymore

File: .devcontainer/scripts/configure-proxies.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (kse3hi) [inline .devcontainer/scripts/configure-proxies.sh:1]: This file will be common for all repos - can we reuse same approach that MS used with https://raw.githubusercontent.com/devcontainers/features/main/src/common-utils/install.sh

**Review by kse3hi — COMMENTED** [review]
See comments

File: requirements.txt (hunk 1 above)
Comment (MP91) [reply requirements.txt:8]: The task was to align the proxy settings, I don't want to mix the stuff here.

File: .devcontainer/scripts/configure-proxies.sh (hunk 3 above)
Comment (MP91) [reply .devcontainer/scripts/configure-proxies.sh:1]: sure, but not in this PR :)

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Pull request merged**